server/util/path/index.js

```diff
@@ -110,6 +110,10 @@ function getExecutablePath(instance_type) {
  */
 function getMediaPaths() {
   var media_paths = [];
+  // User set media_path is first priority
+  if (media_path) {
```

Comment:
Not really, because an empty path would result in the current directory. The standard paths are for platform compatibility. We want to use the standard locations on win, linux and macosx. The standard locations also depends on the linux distribution. At the same time we want to provide support for system-wide / multi-user installations and single-user installations.

* `getMediaPaths()` returns all available media paths
  * These locations are scanned for media repositories
* `media_path` is the default media path
  * This is the location where we create new media repositories
* standardPaths.appDataLocation is a priorized list of locations where a application stores it's data
  * By default we use a subfolder named `media`